samples/hello-world-expo/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "desert-walk-expo-19",
+  "version": "0.0.0",
+  "description": "Hello Expo!",
+  "author": "Jan Aagaard <jan@aagaard.net>",
+  "private": true,
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "web-watch": "webpack --progress --colors --watch"
+  },
+  "dependencies": {
+    "expo": "19.0.1",
+    "react": "16.0.0-alpha.12",
+    "react-dom": "^16.0.0-alpha.12",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-19.0.0.tar.gz",

```

Comment:
Is it possible to use correct react-native version?

Reply:
changed to     "react-native": "0.46.4",
I had some problems on <= 0.44 but newest CLI (expo+react-native) dont care that anymore. Not sure why, but changed!